lib/ValgrindBenchmarkTools/src/ValgrindBenchmarkTools.jl

```diff
@@ -0,0 +1,30 @@
+module ValgrindBenchmarkTools
+
+using Valgrind, Reexport
+@reexport using BenchmarkTools
+
+@inline function prehook()
+    Valgrind.Callgrind.zero_stats()
```

Comment:
You could move this into the `setup_prehook` just so more isolated from the code being benchmarked, i.e. its not part of the function at https://github.com/Zentrik/BenchmarkTools.jl/blob/a77bfca26390681d889460366cb9980ba36103c9/src/execution.jl#L656-L666